_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -109,11 +93,26 @@ This `iframe` element does not have a `tabindex` [attribute value][] that is a n
 <iframe tabindex="0" srcdoc="<a href='/'>Home</a>"></iframe>
 ```
 
+#### Inapplicable Example 2
+
+This `iframe` element contains no content that is part of [sequential focus navigation][].
+
+```html
+<iframe tabindex="-1" srcdoc="<h1>Hello world</h1>"></iframe>
+```
+
+#### Inapplicable Example 3
+
+This `iframe` element contains no [visible][] content because the iframe is hidden.
```

Comment:
The applicability does not mention the `iframe` to be visible or not.

Reply:
I have switched the expectation and applicability around to help make these valid.